INSTALLATION.md

```diff
@@ -112,6 +112,7 @@ This `.env` file must be populated with the following environment variables for
 | RECAPTCHA_SECRET_KEY | Used for authentication using reCAPTCHA                |
 | MAIL_USERNAME        | Used for mailing service                               |
 | MAIL_PASSWORD        | Used for mailing service                               |
+| LAST_RESORT_SUPERADMIN_EMAIL | Used for promoting default super admin         |     
```

Comment:
Provide more information about this parameter. The person installing the app needs to know why it's important not just during installation but also afterwards